src/scrape_up/github/users.py

```diff
@@ -279,3 +279,76 @@ def get_status(self):
         except:
             message = f"Status not found for username {self.username}"
             return message
+    
+    def __get_page_details(self, link):
+        '''
+        scrape the data in the page
+        '''
+        data = requests.get(link)
+        data = BeautifulSoup(data.text, "html.parser")
+        return data
+    
+    def get_pages(self, curr_repo_link, pages_links):
+        '''
+        get the links of all the pages of the repositories
+        '''
+        data = self.__get_page_details(curr_repo_link)
+        pages_body = data.find('div', class_='paginate-container')
+        if pages_body.find('a', class_='next_page') != None:
+            pages_links.append("https://github.com" + pages_body.find('a', class_='next_page')['href'])
+            self.get_pages(pages_links[-1], pages_links)
+
+        return pages_links
+    
+    def repository_stats(self, repo_link):
+        '''
```

Comment:
We already do have methods for these right? in the `Repository` class. Use them.

Reply:
Yes bhaiya, but to avoid creating an object of repo in users class I have included that methods seperately in users class. It will be better than mixing repo class functions in users class ig

Reply:
Better than scraping each page again and again right?

Reply:
I will change bhaiya